File: app/src/main/java/com/hetica/AutismoCordoba/organizar_tareas6.java
package com.hetica.AutismoCordoba;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

/**
 * The type Organizar tareas 6.
 */
public class organizar_tareas6 extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_organizar_tareas6);
    }

    public void onBackPressed(){
        Intent siguiente =new Intent(this, organizar_tareas1.class);
        startActivity(siguiente);
    }

    /**
     * Pasa a la pantalla de elegir cuantas asignaturas vas a estudiar
     *
     * @param view the view
     */
    public void pasar3(View view)
    {
        Intent siguiente =new Intent(this, CuantasAsignaturas.class);
        startActivity(siguiente);
    }

    /**
     * Pasa al anterior consejo
     *
     * @param view the view
     */
    public void pasaratras(View view)
    {
        Intent siguiente =new Intent(this, organizar_tareas1.class);
        startActivity(siguiente);
    }
}

File: app/src/main/java/com/hetica/AutismoCordoba/organizar_tareas6.kt
package com.hetica.AutismoCordoba

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.activity.OnBackPressedCallback
import androidx.appcompat.app.AppCompatActivity

/**
 * The type Organizar tareas 6.
 */
class organizar_tareas6 : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_organizar_tareas6)
        onBackPressedDispatcher.addCallback(this,onBackPressedCallback)

    }

    private val onBackPressedCallback: OnBackPressedCallback = object : OnBackPressedCallback(true) {
        override fun handleOnBackPressed() {
        val siguiente = Intent(this@organizar_tareas6, organizar_tareas1::class.java)
        startActivity(siguiente)
    }}

    /**
     * Pasa a la pantalla de elegir cuantas asignaturas vas a estudiar
     *
     * @param view the view
     */
    fun pasar3(view: View?) {
        val siguiente = Intent(view!!.context, CuantasAsignaturas::class.java)
        startActivity(siguiente)
    }

    /**
     * Pasa al anterior consejo
     *
     * @param view the view
     */
    fun pasaratras(view: View?) {
        val siguiente = Intent(view!!.context, organizar_tareas1::class.java)
        startActivity(siguiente)
    }
}